<launch>
  <!-- Launch the MRS navigation -->
  <node name="grid_map_info" pkg="cola2_turbot" type="grid_map_info" output="screen" respawn="true">
    <rosparam command="load" file="$(find cola2_turbot)/config/MRS/mrs_navigation.yaml"/>
  </node>
  <!-- Launch the grid map visualizer -->
  <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
    <rosparam command="load" file="$(find cola2_turbot)/config/MRS/visualization.yaml"/>
  </node>

</launch>

